apps/daimo-mobile/src/action/useCreateAccount.ts

```diff
@@ -19,12 +19,11 @@ import { defaultEnclaveKeyName, useAccount } from "../model/account";
 export function useCreateAccount(
   name: string,
   inviteLink: DaimoLinkInvite | DaimoLinkNoteV2 | undefined,
-  daimoChain: DaimoChain
+  daimoChain: DaimoChain,
+  keyStatus: DeviceKeyStatus
```

Comment:
i really want to get deeper app logic out of react components 

(and specifically, out of our hooks)

we can use `AccountManager` (or a new `OnboardingManager`) or similar to own this logic instead

OK for now, and it's not rly your fault--hooks just inherently tend toward spaghetti

Reply:
fine for now.

fixing this is a deeper problem. noted here:

https://github.com/daimo-eth/daimo/issues/654